zipkin-server/src/test/java/zipkin2/server/internal/ITZipkinServerSsl.java

```diff
@@ -50,11 +59,13 @@
 @RunWith(SpringRunner.class)
 public class ITZipkinServerSsl {
   @Autowired Server server;
+  @Autowired ArmeriaSettings armeriaSettings;
 
-  // We typically use OkHttp in our tests, but Armeria bundles a handy insecure trust manager
-  final ClientFactory clientFactory = new ClientFactoryBuilder()
-    .sslContextCustomizer(b -> b.trustManager(InsecureTrustManagerFactory.INSTANCE))
-    .build();
+  ClientFactory clientFactory;
+
+  @Before public void configureClientFactory() {
+    clientFactory = configureSsl(new ClientFactoryBuilder(), armeriaSettings.getSsl()).build();
```

Comment:
this was a sneaky way to test the client side without any complex stuff going on, and without using the special armeria context